src/components/layout/organize/SingleCampaignLayout.tsx

```diff
@@ -39,18 +34,18 @@ const SingleCampaignLayout: FunctionComponent<SingleCampaignLayoutProps> = ({ ch
             baseHref={ `/organize/${orgId}/campaigns/${campId}` }
             defaultTab="/"
             fixedHeight={ fixedHeight }
-            subtitle={ startDate && endDate ? (
+            subtitle={ firstEvent && lastEvent ? (
                 <>
                     <FormattedDate
                         day="2-digit"
                         month="long"
-                        value={ startDate }
+                        value={ firstEvent.start_time? removeOffset(firstEvent.start_time) : undefined }
```

Comment:
Can `firstEvent.start_time` be falsy?

Reply:
no - I think I have mistakenly used this as "does firstEvent have a start_time? yes : no", but that is not what it does

Reply:
Yes, that is what it does, but it should always be true. If it's falsy (e.g. `undefined`) something is seriously wrong somewhere.